doc/api/https.md

```diff
@@ -276,6 +307,20 @@ https.createServer(options, (req, res) => {
 }).listen(8000);
 ```
 
+To generate the certificate and key for this example, run:
+
+```bash
+openssl req -x509 -newkey rsa:2048 -nodes -sha256 -subj '/CN=localhost' \
+  -keyout agent2-key.pem -out agent2-cert.pem
```

Comment:
Why adding the `agent2` prefix if they are created ex novo?

Reply:
Oh you're right! I guess I never questioned it, I already had some `.pem` files on the directory where I test the examples one by one from [my previous PR](https://github.com/nodejs/node/pull/54292) so I didn't think about it.

I'll send another commit renaming them to just `private-key` and `certificate`!

Thanks for the review!